package.json

```
@@ -27,14 +27,14 @@
     "eslint-config-prettier": "^8.4.0",
     "eslint-plugin-import": "^2.25.4",
     "eslint-plugin-jquery": "^1.5.1",
-    "eslint-plugin-prettier": "^4.0.0",
+    "eslint-plugin-prettier": "^5.0.0",
```

```suggestion
    "eslint-plugin-prettier": "^5.0.1",
```
Nitpick to match the constraint core uses: https://git.drupalcode.org/project/drupal/-/blob/10.2.x/core/package.json?ref_type=heads#L71